tutorials/polkadot-sdk/build-a-parachain/prepare-relay-chain.md

```diff
@@ -32,7 +32,8 @@ To build a local relay chain, follow these steps:
 1. Clone the most recent release branch of the Polkadot SDK repository to prepare a stable working environment
```

Comment:
```suggestion
1. Clone the most recent release branch of the Polkadot SDK repository to prepare a stable working environment:
```

Reply:
Please make sure items followed by a related terminal command, code snippet, etc. have a colon at the end like I added here. I can't select some of them in this review diff so please check them throughout the document.